libs/common/grpc.go

```diff
@@ -66,14 +73,33 @@ func StartNewGRPCServer(addr string, registerServerHook func(*daprd.Server)) {
 		zlog.Warn().Msg("grpc reflection enabled")
 	}
 
-	zlog.Info().Str("addr", addr).Msg("starting grpc service")
+	interrupted, err := hwutil.RunUntilInterrupted(context.Background(), func() error {
```

Comment:
fyi: Later on we need to pass a caller-defined context here.
https://github.com/helpwave/services/blob/issue/x-spicedb/services/tasks-svc/main.go#L36

Reply:
in pt.2 I already got rid of (most) global logger usage so this would be a [one line change](https://github.com/helpwave/services/pull/606/files#diff-c9f8f86c47f179c3476df0fdb02d379b26c1b903fcb445992bef82e9a23ce3baR44) ig